client/src/pages/transaction.tsx

```diff
@@ -64,7 +64,7 @@ export function Transaction() {
       From: <Code>{data.argument.from}</Code>,
       To: <Code>{data.argument.to}</Code>,
       Amount: token
-        ? `${(data.argument.amount / token.precision).toLocaleString()} ${
+        ? `${(data.argument.amount / 10 ** token.precision).toLocaleString()} ${
```

Comment:
I get the math, but that 10 looks magical to me . . . from where does it come?

Reply:
We store `amount` and `precision` as integers (u64 maybe?) so to send 1 MFX you'd send an amount of 1,000,000,000 knowing that its precision is 9.

Reply:
Wouldn't the base always be 10? Having trouble imagining otherwise.

Reply:
No, you're right, it would be base 10